virttest/utils_test/libvirt.py

```diff
@@ -1532,6 +1532,35 @@ def create_controller_xml(contr_dict):
     return contr
 
 
+def reset_pci_num(vm_name, num=10):
```

Comment:
suggest change default num= 15, since we see "no more pci slot " issue even number is 10